cypress.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = defineConfig({
             email: 'test'+`${randomNumber}`+'@mail.com',
             password: '12345Qwert!',
           };
-        },
+        },     
```

Comment:
Remove the extra spaces after the closing brace.

Reply:
fixed